modular_skyrat/modules/delam_emergency_stop/code/delam.dm

```diff
@@ -0,0 +1,46 @@
+/obj/machinery/power/supermatter_crystal
+	/// If admins and the station have been notified according to the delam suppression function
+	var/station_notified = FALSE
+
+/datum/sm_delam/proc/notify_delam_suppression(obj/machinery/power/supermatter_crystal/sm)
+	if(!sm.is_main_engine)
+		return
+
+	if(sm.station_notified)
+		return
+
+	if(world.time - SSticker.round_start_time > 30 MINUTES)
+		return
+
+	sm.station_notified = TRUE
+	log_admin("DELAM: Round timer under 30 minutes! Supermatter will perform an automatic delam suppression at strength 0%.")
+	message_admins(span_adminnotice("DELAM: Round timer under 30 minutes! [ADMIN_VERBOSEJMP(sm)] will perform an automatic delam suppression at strength 0%. To cancel this, press the 'Toggle Delam Suppression' verb on the EVENTS status tab."))
```

Comment:
Could also be worth just having a `(CANCEL)` href thing like we have for events, here, instead of asking them to use a verb.
```suggestion
	message_admins(span_adminnotice("DELAM: Round timer under 30 minutes! [ADMIN_VERBOSEJMP(sm)] will perform an automatic delam suppression once integrity reaches 0%. To cancel this, press the 'Toggle Delam Suppression' verb on the EVENTS status tab."))
```

Reply:
Ooooh.